CallHome/__init__.py

```diff
@@ -46,32 +47,44 @@ def trn_iter(self):
         annotations = MDTMParser().read(op.join(data_dir, 'callhome.train.mdtm'))
         for uri in sorted(annotations.uris):
             annotation = annotations(uri)
+            uri = 'en_' + uri
+            annotation.uri = uri  # update uri
```

Comment:
Why do we need to add this `en_` prefix?

Reply:
In our data, which seems comes from the original 3 CDs by LDC all file names have prefix `en_` before the speaker Id. So, I assumed the official file name should have `en_` prefix.

Reply:
OK. Thanks.